lib/active_support/cache/memcached_store.rb

```diff
@@ -80,6 +80,12 @@ def initialize(*addresses, **options)
         extend Strategy::LocalCache
       end
 
+      def append(key, value, options = nil)
+        return true if read_only
+        normalized_key = normalize_key(key, options)
```

Comment:
`merged_options` is needed to merge in the default options

```suggestion
        options = merged_options(options)
        key = normalize_key(name, options)
```